app/assets/stylesheets/nav.scss

```diff
@@ -0,0 +1,68 @@
+$neutral: #d9d9d9;
+$poppy: #ec4339;
+$white: #ffffff;
+$grey: #777;
+
+/************** NAVIGATION *******************************/
+
+#nav-container {
+  border-bottom: 1px solid $neutral;
+  box-shadow: 2px 0 5px rgba(0, 0, 0, 0.2);
+}
+
+.dropdown {
+  border: 1px solid $neutral;
+  border-top: none;
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.2);
+}
+
+.dropdown li > a {
+  border-bottom: 1px solid $neutral;
+}
+
+/************** Profile Photo *******************************/
+
+#profile_photo {
+  /*how to change the position of the photo*/
+  width: 30px;
+  height: 30px;
+}
+
+/************** Search Bar *******************************/
+
+#search-bar {
+  width: 115px;
```

Comment:
Properties should be ordered border-color, border-left, border-right, border-top, box-shadow, width